packages/cactus-plugin-keychain-aws-sm/README.md

```diff
@@ -158,7 +158,7 @@ This file lists all the prometheus metrics and what they are used for.
 
 We welcome contributions to Hyperledger Cactus in many forms, and there’s always plenty to do!
 
-Please review [CONTIRBUTING.md](../../CONTRIBUTING.md) to get started.
+Please review [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
```

Comment:
Wow, it was hidden till now XD

Reply:
> Wow, it was hidden till now XD

I was quite surprised by that typo as well :-)